hunting-buddy.lic

```diff
@@ -54,31 +54,36 @@ class HuntingBuddy
       duration = info[:duration]
       stop_on_skills = info['stop_on']
 
-      # Execute scripts to run before the hunt
-      before_actions.each do |before_action|
-        action_parts = before_action.split(' ')
-        script_name = action_parts.shift
-        wait_for_script_to_complete(script_name, action_parts)
-      end
+      #Execute scripts to run before the hunt
+      execute_actions(before_actions)
 
-      next if all_skills_at_cap?(stop_on_skills)
+      if all_skills_at_cap?(stop_on_skills)
```

Comment:
That's a good point.  People seem to be using the before section for various things.  Some are pre-buffing (which isn't needed if the hunt won't start) but some are running athletics or workorders (which should be run), so it's a toss up.

I think that the odds of entering a hunt when all skills are capped is pretty low, which means it makes sense to leave it as it is.